app/src/main/res/values/home_dimens.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
do not use a lot of res files for each project. It's useless. Use one file for all dimensions. Because you are making boilerplate code.

Reply:
done